monitoring/bentoml-monitoring-arize/tests/test_package.py

```diff
@@ -1,8 +1,118 @@
 from __future__ import annotations
 
 import typing as t
-from typing import TYPE_CHECKING
 
 import pytest
+import arize.api
 
-import bentoml
+# from arize.api import Client
+from arize.utils.types import ModelTypes
```

Comment:
```suggestion
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from _pytest.monkeypatch import MonkeyPatch
```

Reply:
Maybe let's just ignore typing for tests?